src/main/java/com/resource/RetrieveTweets.java

```diff
@@ -0,0 +1,37 @@
+package com.resource;
+import com.config.TWConfiguration;
+import org.eclipse.jetty.util.StringUtil;
+import twitter4j.Status;
+import twitter4j.Twitter;
+import twitter4j.TwitterException;
+import twitter4j.TwitterFactory;
+import twitter4j.conf.ConfigurationBuilder;
+import java.util.List;
+public class RetrieveTweets {
+    public  static  String[] latestTweet(){
```

Comment:
Try to return List<String> instead of Array of String 
https://www.geeksforgeeks.org/list-interface-java-examples/
Please refer above link for examples

Reply:
now returning object of ArrayList<String>.